yapapi/executor/__init__.py

```diff
@@ -341,27 +341,15 @@ async def start_worker(agreement: rest.market.Agreement) -> None:
                             async for evt_ctx in remote:
                                 evt = evt_ctx.event(agr_id=agreement.id, task_id=task_id, cmds=cmds)
                                 emit(evt)
+                                if isinstance(evt, events.CommandExecuted) and not evt.success:
+                                    raise CommandExecutionError(evt.command, evt.message)
 
                             emit(events.GettingResults(agr_id=agreement.id, task_id=task_id))
                             await batch.post()
                             emit(events.ScriptFinished(agr_id=agreement.id, task_id=task_id))
                             await accept_payment_for_agreement(agreement.id, partial=True)
 
-                        except Exception as err:
-
-                            if isinstance(err, CommandExecutionError):
```

Comment:
We no longer need to catch `CommandExecutionError` flying by just to emit `CommandExecuted` event with `success=False` and re-raise the exception, the even is emitted correctly in line 343 above.